doc/modules/linear_model.rst

```diff
@@ -136,17 +136,27 @@ Setting the regularization parameter: generalized Cross-Validation
 ------------------------------------------------------------------
 
 :class:`RidgeCV` implements ridge regression with built-in
-cross-validation of the alpha parameter.  The object works in the same way
+cross-validation of the alpha parameter. The object works in the same way
 as GridSearchCV except that it defaults to Generalized Cross-Validation
 (GCV), an efficient form of leave-one-out cross-validation::
 
     >>> from sklearn import linear_model
-    >>> reg = linear_model.RidgeCV(alphas=[0.1, 1.0, 10.0], cv=3)
-    >>> reg.fit([[0, 0], [0, 0], [1, 1]], [0, .1, 1])       # doctest: +SKIP
-    RidgeCV(alphas=[0.1, 1.0, 10.0], cv=3, fit_intercept=True, scoring=None,
-        normalize=False)
-    >>> reg.alpha_                                      # doctest: +SKIP
-    0.1
+    >>> reg = linear_model.RidgeCV(alphas=[1.0, 10.0], store_cv_values=True)
+    >>> reg.fit([[0, 0], [0, 0], [1, 1]], [0, .1, 1])       # doctest: +NORMALIZE_WHITESPACE
+    RidgeCV(alphas=array([ 1., 10.]), cv=None, fit_intercept=True, gcv_mode=None,
+            normalize=False, scoring=None, store_cv_values=True)
+    >>> reg.alpha_
+    1.0
+    >>> reg.cv_values_                                      # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+    array([[0.105...   , 0.259...],
+           [0.022...   , 0.125...],
+           [0.902...   , 0.902...]])
```

Comment:
Maybe we should not mention the `store_cv_values` / `cv_values_` things in the main documentation if we are not 100% satisfied with the current state of affairs (e.g. memory usage with large `n_samples` and handling of the sample weights). Let's keep this for another PR.

+1 for keeping `reg.alpha_` in the inline example though. I think you should even demonstrate a larger range for alpha, e.g. `alphas=np.logspace(-6, 6, 13)`.